data_files/IBMZ-container-registry.json

```diff
@@ -0,0 +1,672 @@
+  },
+  {
+    "packageName": "compose",
+    "version": "2.0.1",
+    "description": "Compose is a tool for defining and running multi-container Docker applications."
+  },
+  {
+    "packageName": "compose",
+    "version": "2.2.3",
+    "description": "Compose is a tool for defining and running multi-container Docker applications."
+  },
+  {
+    "packageName": "couchdb",
+    "version": "3.1.1",
+    "description": "Apache CouchDB is an open-source document-oriented NoSQL database, implemented in Erlang."
+  },
+  {
+    "packageName": "debian",
+    "version": "bullseye",
+    "description": "Debian is an operating system which is composed primarily of free and open-source software, most of which is under the GNU General Public License, and developed by a group of individuals known as the Debian project."
+  },
+  {
+    "packageName": "debian",
+    "version": "bullseye-slim",
+    "description": "Debian is an operating system which is composed primarily of free and open-source software, most of which is under the GNU General Public License, and developed by a group of individuals known as the Debian project."
+  },
+  {
+    "packageName": "debian",
+    "version": "buster",
+    "description": "Debian is an operating system which is composed primarily of free and open-source software, most of which is under the GNU General Public License, and developed by a group of individuals known as the Debian project."
+  },
+  {
+    "packageName": "debian",
+    "version": "buster-slim",
+    "description": "Debian is an operating system which is composed primarily of free and open-source software, most of which is under the GNU General Public License, and developed by a group of individuals known as the Debian project."
+  },
+  {
+    "packageName": "debian",
+    "version": "sid",
+    "description": "Debian is an operating system which is composed primarily of free and open-source software, most of which is under the GNU General Public License, and developed by a group of individuals known as the Debian project."
+  },
+  {
+    "packageName": "debian",
+    "version": "sid-slim",
+    "description": "Debian is an operating system which is composed primarily of free and open-source software, most of which is under the GNU General Public License, and developed by a group of individuals known as the Debian project."
+  },
+  {
+    "packageName": "debian",
+    "version": "stable",
+    "description": "Debian is an operating system which is composed primarily of free and open-source software, most of which is under the GNU General Public License, and developed by a group of individuals known as the Debian project."
+  },
+  {
+    "packageName": "debian",
+    "version": "stable-slim",
+    "description": "Debian is an operating system which is composed primarily of free and open-source software, most of which is under the GNU General Public License, and developed by a group of individuals known as the Debian project."
```

Comment:
Also, should debian be part of the package list? It has several versions too

Reply:
These are container images, these ones are just basic Debian images :+1: